ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -2992,6 +2996,8 @@
 				GCC_NO_COMMON_BLOCKS = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				PRODUCT_BUNDLE_IDENTIFIER = "org.reactivecocoa.$(PRODUCT_NAME:rfc1034identifier)-Tests";
+				PRODUCT_NAME = "$(PROJECT_NAME)";
```

Comment:
Which should be OK? `ReactiveCocoa.framework` and `ReactiveCocoa.xctest`, right? I can't get the dependencies to build with Xcode 7.1 :(